subworkflows/nf-core/fastq_contam_seqtk_kraken/main.nf

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nextflow
+
+//
+// FASTQ_CONTAM_SEQTK_KRAKEN: Subsample FASTQs and perform contamination screening
+//
+include { KRAKEN2_KRAKEN2 as KRAKEN2 } from '../../../modules/nf-core/kraken2/kraken2/main'
+include { SEQTK_SAMPLE               } from '../../../modules/nf-core/seqtk/sample/main'
+include { UNTAR as UNTAR_KRAKEN2_DB  } from '../../../modules/nf-core/untar/main'
```

Comment:
Do we need to rename `UNTAR` as `UNTAR_KRAKEN2_DB`?
Since `UNTAR` would be in `FASTQ_CONTAM_SEQTK_KRAKEN` subworkflow.

Reply:
Actually, where is UNTAR in this subworkflow?

Reply:
good point 😅

Reply:
I removed it, should ditch it here too 👍🏻 Wasn't sure if this is something we have to handle in the actual workflow logic to untar stuff there (probably better) and just assume untarred input 👍🏻

Reply:
Ditched it, thanks @maxulysse !